docs/blog/20230701-Daeraxa-JulyUpdate.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Community Update
+author: Daeraxa
+date: 2023-07-01
+category:
+  - news
+  - log
+tag:
+  - update
+---
+
+What has it got in its pocketses? It's the July community update!
+
+<!-- more -->
+
+# Welcome to the July Community Update
+
+This month's update is a maybe a little different in tone to some of the others in that we have had to deal with some problems recently outside of our control which means our first few items are a little more "serious" and a bit less optimistic that our usual updates. That being said these have been front loaded into this update with all the good news coming in the second half, so don't despair!
+
+This month we have to cover some of the issues such as our CI not building binaries, discussions about the future of our subreddit and some false antivirus positives on our packages but we then come back strong with a new method of downloading Pulsar and a bunch of improvements to the application!
```

Comment:
Again I'll pick from this as I kind of abhor the "Oxford comma" unless being used to remove ambiguity. I'm also really not keen on starting sentences with "but", I know it isn't a hard rule but in this case I find it kind of jarring.
I also think it should be restructured to allow "cleanse the palate" rather than "cleans".